src/components/ThemeModeSwitcher/index.jsx

```diff
@@ -0,0 +1,41 @@
+/* Temporary component for testing theme implementation and dark mode.  Used in the right side
+nav menu currently.
+
+@TODO convert to some final implementation
+*/
+import { useEffect, useState } from 'react';
+import {
+  //   Experimental_CssVarsProvider as CssVarsProvider,
+  useColorScheme,
+} from '@mui/material/styles';
+import styles from './modeSwitcher.module.css';
+
+export const ModeSwitcher = () => {
+  const { mode, setMode } = useColorScheme();
+  const [mounted, setMounted] = useState(false);
+
+  //Need to figure out how to properly reference the theme provided by the CssVarsProvider context.
+  console.log('Called mode switcher');
+
+  useEffect(() => {
+    setMounted(true);
+  }, []);
+  if (!mounted) {
+    return null;
+  }
```

Comment:
```suggestion
  useEffect(() => {
    setMounted(true);
  }, []);
  if (!mounted) {
    return null;
  }
  ```
  In what case will this component not be mounted?

Reply:
Probably never, that mode switch button was just copied from the MUI docs, so it's probably designed to cover more cases than we have...  I'll remove the useEffect